code/controllers/subsystems/jobs.dm

```diff
@@ -395,6 +395,12 @@ SUBSYSTEM_DEF(job)
 		C.install_default_modules_by_job(job)
 		C.access.Add(job.cruciform_access)
 
+	var/obj/item/weapon/oddity/secdocs/D
+	if(D.inv_spawn_count > 0 && prob(5) && !(locate(/obj/item/weapon/oddity/secdocs) in H.get_contents()))
```

Comment:
Trying to read a var off empty `D` looks dysfunctional and/or cursed. Which one is it?

Reply:
The latter, it is a static variable.